skills/findChannel.js

```diff
@@ -0,0 +1,21 @@
+class Project{
```

Comment:
I don’t think `/skills` is the best place for this. Maybe talk to Nathan and Colby about the right place for things like this.

Reply:
ok perfect. Nathan told me to put it in a lib file